dronegen/buildbox.go

```diff
@@ -29,12 +29,8 @@ func buildboxPipelineSteps() []step {
 		waitForDockerStep(),
 	}
 
-	for _, name := range []string{"buildbox", "buildbox-arm"} {
+	for _, name := range []string{"buildbox", "buildbox-arm", "buildbox-centos7"} {
 		for _, fips := range []bool{false, true} {
-			// FIPS is unsupported on ARM/ARM64
-			if name == "buildbox-arm" && fips {
-				continue
-			}
```

Comment:
What changed here to allow FIPS on ARM?

Reply:
Nothing currently, however the Dockerfile and Makefile targets exist so I added the ability for drone to keep the image up-to-date. The buildbox isn't used yet. I can revert this change if desired, but it doesn't actually start building images just keeps the buildbox image up to date.

Reply:
I went ahead and removed the buildbox-arm + fips combination and kept it to the original.